lib/mind_the_gapp/mta/scheduled_arrival_time.ex

```diff
@@ -0,0 +1,45 @@
+defmodule MindTheGapp.Mta.ScheduledArrivalTime do
```

Comment:
This seems like it is more of a utility to load a static file than it is about working with ScheduledArrivalTimes in our application. I wonder if it should be named as such?

Reply:
I agree.